README.md

```diff
@@ -30,21 +30,235 @@
     </a>
 </p>
 
-<p align="center">
-    Train and deploy models to make online predictions using only SQL, with an open source extension for Postgres. Manage your projects and visualize datasets using the built-in dashboard.
-</p>
 
-![PostgresML in practice](pgml-docs/docs/images/console.png)
+## Table of contents
+- [Introduction](#introduction)
+- [Installation](#installation)
+- [Getting started](#getting-started)
+- [Natural Language Processing](#nlp-tasks)
+- [Regression](#regression)
+- [Classification](#classification)
 
-The dashboard makes it easy to compare different algorithms or hyperparameters across models and datasets.
+## Introduction
+PostgresML is a PostgreSQL extension that enables you to perform ML training and inference on text and tabular data using SQL queries. With PostgresML, you can seamlessly integrate machine learning models into your PostgreSQL database and harness the power of cutting-edge algorithms to process text and tabular data efficiently.
 
-[![PostgresML dashboard](pgml-docs/docs/images/dashboard/models.png)](https://cloud.postgresml.org/)
+### Text Data
+- Perform natural language processing (NLP) tasks like sentiment analysis, question and answering, translation, summarization and text generation
+- Access 1000s of state-of-the-art language models like GPT-2, GPT-J, GPT-Neo from :hugs: HuggingFace model hub
+- Fine tune large language models (LLMs) on your own text data for different tasks
 
-<h2 align="center">
-    See it in action — <a href="https://cloud.postgresml.org/" target="_blank">cloud.postgresml.org</a>
-</h2>
+**Translation**
+<table>
```

Comment:
These tables are wider than max width and create a scrollbar. You could use another <thead><row> for the result, or else we should shorten the content to avoid bars.